libs/blocks/tabs/tabs.js

```diff
@@ -243,9 +243,9 @@ const init = (block) => {
   }
 
   // Tab Paddles
-  const tabPaddles = createTag('div', { class: 'tab-paddles', role: 'scrollbar' });
-  const paddleLeft = createTag('button', { class: 'paddle paddle-left', disabled: '' }, RIGHT_PADDLE);
-  const paddleRight = createTag('button', { class: 'paddle paddle-right', disabled: '' }, RIGHT_PADDLE);
+  const tabPaddles = createTag('div', { class: 'tab-paddles', role: 'scrollbar', 'aria-valuenow': 0 });
```

Comment:
Documentation states:
> If there is no known value, like when a progress bar is in an indeterminate state, don't set an aria-valuenow attribute.

Is there a value we could assign? Otherwise this might need a bit more investigation.

Reply:
Yes, we can assign a value. I added an event listener to calculate the scroll position with each scroll event.
Thanks for pointing it out 👍